src/main/java/io/specto/hoverfly/junit/verification/InOrderVerification.java

```diff
@@ -0,0 +1,54 @@
+package io.specto.hoverfly.junit.verification;
+
+import io.specto.hoverfly.junit.api.HoverflyClient;
+import io.specto.hoverfly.junit.core.model.JournalEntry;
+import io.specto.hoverfly.junit.dsl.RequestMatcherBuilder;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+import static io.specto.hoverfly.junit.verification.HoverflyVerifications.times;
+
+public class InOrderVerification {
+
+
+    private HoverflyClient hoverflyClient;
+    private List<RequestMatcherBuilder> requestMatchers = new ArrayList<>();
+
+    public InOrderVerification(HoverflyClient hoverflyClient, RequestMatcherBuilder... requestMatchers) {
+        this.hoverflyClient = hoverflyClient;
+        this.requestMatchers.addAll(Arrays.asList(requestMatchers));
+    }
+
+    public void verify() {
```

Comment:
Ok, will leave this out from the current release, and wait until a more advanced verification logic is implemented in Hoverfly